File: API/Data/Models/OrderAgregate/order.cs
using System;
using System.Collections.Generic;
using System.Reflection.Metadata.Ecma335;
using API.Data.Models.OrderAgregate;
using EStore.API.Data.Models;

public class order
{
    public int Id { get; set; }
    public int Company_Id { get; set; }
    public company Company { get; set; }
    public string BuyerId { get; set; }
    public shippingAdress ShippingAdress { get; set; }
    public DateTime OrderDate { get; set; } = DateTime.Now;
    public List<orderItem> OrderItems { get; set; }
    public decimal SubTotal { get; set; }
    public decimal DeliveryFee { get; set; }
    public orderStatus OrderStatus { get; set; } = orderStatus.Pending;

    public decimal GetTotal()
    {
        return SubTotal + DeliveryFee;
    }
}

File: client/src/app/models/order.ts
export interface ShippingAdress {
  fullName: string;
  adress1: string;
  google_Map: string;
  city: string;
  state: string;
  phone_Number: string;
  eMail: string;
}

export interface OrderItem {
  itemId: number;
  name: string;
  pictureUrl: string;
  price: number;
  quantity: number;
}

export interface Order {
  id: number;
  company_Id: number;
  company?: any;
  buyerId: string;
  shippingAdress: ShippingAdress;
  orderDate: string;
  orderItems: OrderItem[];
  subTotal: number;
  deliveryFee: number;
  orderStatus: string;
  total: number;
}